### PR TITLE
fix: Approve UX 

### LINKED ISSUE #11: UX issue: It's not clear which tags are saved on tree rejection
The *Tags* sidebar on the *Verify* page has a set of tag options, some of which fall under *Approve* and *Reject* headings.

What's not clear is that when you reject a tree, only the `rejection_reason` is recorded.
All other tags (`morphology`, `age` and `species_id`) are discarded, even though the inputs are still enabled and visible.

Perhaps all tag inputs should sit under either *Approve* or *Reject*.

tags

reject

### PR DESCRIPTION
**Pull request by luacmartins**
Resolves #11.

@nmcharlton Is this the type of functionality we are looking for? If not, I will be glad to make changes.

### CODE REVIEW

File: ..env.development.local [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_WEBMAP_DOMAIN=http://dev.treetracker.org
```
Comment (nmcharlton) [inline ..env.development.local:1]: I don't think you meant to commit this 😉

File: src/components/TreeImageScrubber.js [hunk 2]
```diff
@@ -767,59 +710,135 @@ function SidePanel(props) {
...
+          </RadioGroup>
+        </Grid>
+        */}
+              <Grid>
+                <Typography className={classes.sidePanelItem} variant="h6">
+                  Species (if known)
+                </Typography>
+                <Species />
+              </Grid>
+              <Grid>
```
Comment (nmcharlton) [inline src/components/TreeImageScrubber.js:776]: The _Additional tags_ should be available for both _Approve_ and _Reject_ operations.

**Comment by nmcharlton** [comment]
> @nmcharlton Is this the type of functionality we are looking for? If not, I will be glad to make changes.

@luacmartins Yep, this is exactly what I had in mind 😃

File: ..env.development.local (hunk 1 above)
Comment (luacmartins) [reply ..env.development.local]: Oops! Good catch, thanks!

File: src/components/TreeImageScrubber.js (hunk 2 above)
Comment (luacmartins) [reply src/components/TreeImageScrubber.js:779]: Makes sense. I added them to both now!

**Review by nmcharlton — APPROVED** [review]
Perfect, thanks @luacmartins!

**Pull request merged**